ui/src/cloud/selectors/demodata.ts

```diff
@@ -0,0 +1,11 @@
+import {get, differenceBy} from 'lodash'
+import {AppState, Bucket, DemoBucket} from 'src/types'
+
+export const getNewDemoBuckets = (state: AppState, ownBuckets: Bucket[]) => {
+  const demoDataBuckets = get(
+    state,
+    'cloud.demoData.buckets',
+    []
+  ) as DemoBucket[]
+  return differenceBy(demoDataBuckets, ownBuckets, b => b.id)
```

Comment:
is there an explicit default sort that should be applied here?

Reply:
will alphabetize